src/client.ts

```diff
@@ -2242,6 +2215,40 @@ export class StreamChat<
     return false;
   }
 
+  /**
+   * channelMuteStatus - check if the channel is muted or not, can be used after connectUser() is called
+   *
+   * @param cid
+   */
+  channelMuteStatus = (cid: string) => this._muteStatus(cid).muted;
```

Comment:
We have a similar function for `userMuteStatus`!! So makes sense to have this function as well